openssl-sys/build.rs

```diff
@@ -15,10 +15,13 @@ fn main() {
         // rustc doesn't seem to work with pkg-config's output in mingw64
         if !target.contains("windows") {
             if let Ok(info) = pkg_config::find_library("openssl") {
-                // avoid empty include paths as they are not supported by GCC
+                // if we did not find any include paths try to find some other
+                // ones we might want to emit.
                 if info.include_paths.len() > 0 {
                     let paths = env::join_paths(info.include_paths).unwrap();
                     println!("cargo:include={}", paths.to_str().unwrap());
+                } else {
+                    find_fallback_headers();
```

Comment:
I'm not sure if it'll come up, but I'm a bit paranoid, since a couple of functions *did* change their signatures and ABIs from 0.9.8 to 1.0.0 and openssl-sys-extras relies on the correctness of these headers to handle that.